frontend/src/scenes/AdminPage.tsx

```diff
@@ -215,39 +238,55 @@ function UserTabContent() {
     }
   }, [adminService]);
 
+  const columns: ColDef[] = [
+    { field: 'id', headerName: 'ID', width: 80 },
+    { field: 'name', headerName: 'Name', width: 300 },
+    { field: 'phoneNumber', headerName: 'Phone Number', width: 150 },
+    { field: 'destinationCountry', headerName: 'Country' },
+    { field: 'callTime', headerName: 'Call Time Balance', width: 300 },
+    {
+      field: 'transactionHistory',
+      headerName: 'Transaction History',
+      width: 400,
+      renderCell(rowData) {
+        return (
+          <Link to={`${PATHS.TRANSACTIONS}?user=${rowData.value}`}>View</Link>
+        );
+      },
+    },
+  ];
+
+  const rows: {
+    id: any;
```

Comment:
Try defining the row's type as an Interface and avoiding the use of `any` where possible.